package.json

```diff
@@ -48,9 +48,8 @@
     "@apostrophecms-pro/cypress-tools": "1.0.0-beta.19",
     "autoprefixer": "^10.4.20",
     "cypress": "^13.16.0",
-    "eslint": "^8.0.0",
-    "eslint-config-apostrophe": "^4.3.0",
-    "eslint-plugin-cypress": "^2.15.2",
+    "eslint-config-apostrophe": "github:apostrophecms/eslint-config-apostrophe#pro-8109-upgrade-dependencies",
```

Comment:
TODO: use version 6 when https://github.com/apostrophecms/eslint-config-apostrophe/pull/22 is merged